hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieCompactionMetadata.java

```diff
@@ -51,6 +52,14 @@ public void addWriteStat(String partitionPath, CompactionWriteStat stat) {
     partitionToCompactionWriteStats.get(partitionPath).add(stat);
   }
 
+  public void setCompactionCommit(long compactionCommit) {
```

Comment:
this is hacky, trying to separate out the compactionCommit out of the `RealtimeCompactor` once we have the compact() in the hoodieClient.

Reply:
Don't follow this change, that well tbh..